src/ui/nqqrun.cpp

```diff
@@ -304,9 +320,27 @@ void RunDialog::slotSave()
             &ok);
     if (ok && !name.isEmpty() && !m_command->text().isEmpty()) {
         m_settings.Run.setCommand(name, m_command->text());
+        m_saved = 1;
+
+        //Graphical fade for save state
+        m_status->setText(tr("Command saved..."));
+        QGraphicsOpacityEffect *eff = new QGraphicsOpacityEffect(m_status);
+        m_status->setGraphicsEffect(eff);
+        QPropertyAnimation *a = new QPropertyAnimation(eff, "opacity");
+        a->setDuration(10000);
+        a->setStartValue(5);
+        a->setEndValue(0);
+        a->setEasingCurve(QEasingCurve::Linear);
+        a->start(QPropertyAnimation::DeleteWhenStopped);
+        connect(a, SIGNAL(finished()), this, SLOT(slotHideStatus()));
```

Comment:
for uniformity, use the new syntax for signal/slots: https://wiki.qt.io/New_Signal_Slot_Syntax